### PR TITLE
React Calculator Milestone 3: Styling!

### PR DESCRIPTION
**Pull request by lmaldonadoch**
For this milestone, I added styling to the calculator. I added the properties to the Button class and passed them through the ButtonPannel class so that it would have the necessary information to render the layout correctly.

## Please note!
ESlynt was complaining about 2 violations. I disabled them and here are the reasons:

## Module-specific requirements.

- Make sure that you use props and state in a way they were designed for.
- Make sure that you understand the difference between stateful and stateless components. 

## Project Specific

- Use flex display mode for App, Display and ButtonPanel components
  * The Calculator width should be 700px.
- Style the Display component
  * It should have a gray background.
  * It should have a height set to 100px.
  * The result should be presented with a white and bold text.
  * The result should have padding from the edges.
  * The result should be aligned to the right.
- Style the ButtonPanel component
  * Each row (display and group buttons) should have a height of 100 px.
  * The Button groups should be displayed horizontally in rows.
- Style the Button component
  * Button should take 25% of the container width, except for the button that represents the “0” (zero).
  * The text in the buttons should be centered and displayed in black color.
  * Each button should have a border.
- Pass “color” and “wide” props to the Button
  * The color prop should be used to customize the color of the button. If it is not passed, the button should have an orange background by default.
  * The wide prop should accept a boolean to indicate that the button should have twice the standard width (used for the “0” button).

## General requirements:
- Add a descriptive README file to your repositories as described.
- Use meaningful names for your Pull Requests and add a summary of its content as described.
- Use descriptive commit messages as described.
- Do not commit files that are not related to the project. Instead, add them to .gitignore file.
- Do not commit inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

### Git Flow:
- Use a specific branch for each feature.
- Use a descriptive name for your branches.
- Use Rubocop to validate your code.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    );
+  }
+}
+
+Button.propTypes = {
+  name: PropTypes.string.isRequired,
+  wide: PropTypes.string.isRequired,
+  color: PropTypes.string,
+};
+
```
Comment (shloch) [inline src/components/Button.js]: `wide` should be `Boolean` 
https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png

File: src/components/ButtonPanel.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import Button from './Button';
+
+export default class ButtonPanel extends React.Component {
+  renderButton(name) {
+    const wide = name === '0' ? '50%' : '25%';
+    const color = !['÷', 'X', '-', '+', '='].includes(name)
+      ? 'light-grey'
+      : 'orange';
+    return <Button name={name} wide={wide} color={color} />;
+  }
```
Comment (shloch) [inline src/components/ButtonPanel.js]: meaning you'll tweak this `if` statement a little regarding `wide` property which should be boolean

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @lmaldonadoch , 

Nice job really,
Logic as it is works properly but for the sake of sticking with guidelines, we have a small modification to do (mentioned below) :

File: src/components/Button.js (hunk 1 above)
Comment (lmaldonadoch) [reply src/components/Button.js]: Thanks for pointing it out! Is it solved!

File: src/components/ButtonPanel.js (hunk 2 above)
Comment (lmaldonadoch) [reply src/components/ButtonPanel.js]: Thanks for pointing it out! Is it solved!

File: src/components/Button.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ export default class Button extends React.Component {
 Button.propTypes = {
   name: PropTypes.string.isRequired,
   wide: PropTypes.string.isRequired,
-  color: PropTypes.string,
+  color: PropTypes.bool,
```
Comment (Oghenebrume50) [inline src/components/Button.js]: What you want to make boolean here is the wide property not the color

File: src/components/ButtonPanel.js [hunk 4]
```diff
@@ -5,7 +5,7 @@ import Button from './Button';
 
 export default class ButtonPanel extends React.Component {
   renderButton(name) {
-    const wide = name === '0' ? '50%' : '25%';
+    const wide = name === '0' ? true : false;
```
Comment (Oghenebrume50) [inline src/components/ButtonPanel.js]: If the wide is boolean you could just do `wide ?  '50%' : '25%' `

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED

Hi @lmaldonadoch good job working on this you have done a lot with this project and thanks for adhering to the previous review :+1:  however I noticed a little mix-up and I have pointed it out below, please look inot those and make some changes. :slightly_smiling_face: 

**Please request for another review, after making changes** :slightly_smiling_face: 

**RAPHAEL NORIODE** :slightly_smiling_face:

File: src/components/Button.js (hunk 3 above)
Comment (lmaldonadoch) [reply src/components/Button.js]: Thanks for pointing it out, it is resolved!

File: src/components/ButtonPanel.js (hunk 4 above)
Comment (lmaldonadoch) [reply src/components/ButtonPanel.js]: Thanks for pointing it out, it is resolved!

**Review by bellom — APPROVED** [review]
## Hi @xtrmdarc :clap: 

##### You have done wonderfully well, you passed all checklist for this milestone :+1:
 
## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck with your next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

**Pull request merged**